src/scss/jooa11y.scss

```diff
@@ -620,10 +639,10 @@ html[data-jooa11y-theme="dark"] {
   background: none;
   border: 0;
   border-radius: 50%;
-  transition: all .2s ease-in-out;
+  transition: all 0.2s ease-in-out;
 }
 
-[dir="rtl"] #jooa11y-container .jooa11y-close-btn {
+[dir=rtl] #jooa11y-container .jooa11y-close-btn {
```

Comment:
no thanks - the quotes should be used
https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors

Reply:
strange there I did nothing. Maybe there is something happend  I do not control. I have to check.